### PR TITLE
parallel-doc

### PR DESCRIPTION
**Pull request by saikrishna321**
## Proposed changes

Updated parallel doc as per appium v 1.7.0

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/en/advanced-concepts/parallel_tests.md [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
+#### Simulator
 
-Unfortunately, running local parallel iOS tests isn't currently possible. Unlike Android, only one version of the iOS simulator can be launched at a time, making it run multiple tests at once.
+- `udid` simulator UDID,this can be retrieved from xcrun simctl list.
+- `deviceName` Simulator Name
+- `platformVersion` Simulator OS version
+- `wdaLocalPort` unique wdaPort, as WDA defaults to 8100
 
-If you do want to run parallel iOS tests, you need to use Sauce. Simply upload your Appium test to Sauce, and it can run as many parallel iOS or Android tests as your account allows. See more about running your tests on Sauce [here](https://docs.saucelabs.com/tutorials/appium/).
+Parallel Safari/Webview sessions are not working due to an Apple bug
```
Comment (mykola-mokhnach) [inline docs/en/advanced-concepts/parallel_tests.md]: They don't work on Simulator, but real device should be OK by providing _webkitDebugProxyPort_ capability.

Comment (saikrishna321) [reply docs/en/advanced-concepts/parallel_tests.md]: Done

**Review by KazuCocoa — APPROVED** [review]
looks good to me 👍

File: docs/en/advanced-concepts/parallel_tests.md [hunk 2]
```diff
@@ -1,32 +1,37 @@
...
-### Parallel iOS Tests
+#### Parallel Safari Real device
 
-Unfortunately, running local parallel iOS tests isn't currently possible. Unlike Android, only one version of the iOS simulator can be launched at a time, making it run multiple tests at once.
+- `udid` the device id
+- `wdaLocalPort` unique wdaPort, as WDA defaults to 8100
+- `webkitDebugProxyPort` unique webKitProxy, as IWDP defaults to 27753
 
-If you do want to run parallel iOS tests, you need to use Sauce. Simply upload your Appium test to Sauce, and it can run as many parallel iOS or Android tests as your account allows. See more about running your tests on Sauce [here](https://docs.saucelabs.com/tutorials/appium/).
+Parallel Safari/Webview sessions are not working due to an Apple bug
```
Comment (mykola-mokhnach) [inline docs/en/advanced-concepts/parallel_tests.md]: I assume this phrase has to be moved to the above category

Comment (saikrishna321) [reply docs/en/advanced-concepts/parallel_tests.md]: Done

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by saikrishna321** [comment]
@imurchie ping

**Review by SrinivasanTarget — APPROVED** [review]
👍

**Review by imurchie — APPROVED** [review]

**Review by jlipps — COMMENTED** [review]
good doc updates. We should probably link to this from the Create Session command docs as well.

**Pull request merged**